test/upgrade/prober/wathola/event/services.go

```diff
@@ -26,17 +26,27 @@ import (
 var mutex = sync.RWMutex{}
 var lastProgressReport = time.Now()
 
+const (
+	TagDuplicated = "DUPLICATE"
+	TagDefault    = "DEFAULT"
+	TagMissing    = "MISSING"
+)
```

Comment:
Remove those tags. We can live without them. Better to have more methods, then additional untyped, unbounded parameters to those methods, and a bunch of if statements in them.

Reply:
Removed.